.buildkite/commands/checkout-release-branch.sh

```diff
@@ -2,16 +2,8 @@
 
 echo "--- :git: Checkout Release Branch"
 
-# Note: `BUILDKITE_RELEASE_VERSION` is the legacy environment variable passed to Buildkite by ReleaseV2.
-# It used the `BUILDKITE_` prefix so it was not filtered out when passed to the MacOS VMs, due to how `hostmgr` works.
-# This is considered legacy: we should eventually remove all use of custom `BUILDKITE_` variables, and instead
-# resolve the value of those sooner (i.e. in the YML pipeline) then pass it as parameter to the `.sh` calls instead.
-
-# Use the provided argument if there's one, otherwise fall back to the legacy BUILDKITE_RELEASE_VERSION
-RELEASE_VERSION=${1:-$BUILDKITE_RELEASE_VERSION}
-
 if [[ -z "${RELEASE_VERSION}" ]]; then
-    echo "RELEASE_VERSION is not set and BUILDKITE_RELEASE_VERSION is not available."
+    echo "RELEASE_VERSION is not set."
     exit 1
 fi
```

Comment:
This will lead to [the same issue I mentioned in the `DayOne-Apple` PR counterpart](https://github.com/bloom/DayOne-Apple/pull/24982/files#r1838530079), i.e:
 - You don't read the value from `$1` anymore
 - And also reading the value of an undefined or empty env var will crash the script before it even have time to execute the `then` clause given `bash -eu`

The most concise fix for this would be to use the `${1:?error message}` syntax to kill two birds with one stone:
```suggestion
RELEASE_VERSION="${1:?RELEASE_VERSION parameter missing}"
```

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/14381/commits/2516ea76dc0822b86d5d50529167de4eeedf8936. Thank you @AliSoftware!